OWNERS.md

```diff
@@ -67,7 +67,7 @@ The following extension maintainers have final say over the extensions mentioned
 approve an extension PR, it will be merged by the maintainer on-call (or any other maintainer)
 without further review.
 
-* Piotr Sikora ([PiotrSikora](https://github.com/PiotrSikora)) (piotrsikora@google.com)
+* Piotr Sikora ([PiotrSikora](https://github.com/PiotrSikora)) (psikora@aviatrix.com)
```

Comment:
Actually, it should be `piotr@aviatrix.com`... but I no longer work on Wasm in Envoy and @mpwarres should replace me here on behalf of Google, since he took over those responsibilities.

Reply:
Ah, thanks!